### DISCUSSION TITLE
Invalid Configuration, CNAME Record already set

### DISCUSSION THREAD

**Discussion by motch26** [vercel/community#2197] [Help]
### Summary

I have already set a CNAME Record and is already propagated but the configuration is still invalid.
Can please someone help. Thanks.

Hosting: GoDaddy

![image](https://user-images.githubusercontent.com/84578100/234556607-197ff9cf-bb47-41d5-a5ef-085663d4f452.png)
![image](https://user-images.githubusercontent.com/84578100/234556694-7d442e15-e9b3-4df4-925c-1527d8fc0118.png)

**Comment by sostenesapollo** [comment]
I'm dealing with the same error, do you get some update ?

**Comment by amyegan** [comment]
It can take 24-48 hours for changes to fully propagate, so it may be working now. 🙏 Are you still having trouble with your domain? 

You can check DNS propagation status using a tool like [whatsmydns.net](https://www.whatsmydns.net/) or `dig`. More troubleshooting info can be found in our docs:
- [Why is my Vercel domain not verified?](https://vercel.com/guides/why-is-my-vercel-domain-unverified)
- [DNS propagation](https://vercel.com/docs/concepts/projects/domains#dns-propagation)
- [Troubleshooting domains](https://vercel.com/docs/concepts/projects/domains/troubleshooting#)
